readme.md

```diff
@@ -44,6 +44,11 @@ You will need to build assets the first time (you will need [Yarn](https://yarnp
 $ yarn
 $ ./node_modules/gulp/bin/gulp.js --production
 ```
+Now setup a 32 character application key that will be used by the app (following only works on unix):
+
+```
+export APP_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
@sporchia  Think that's preferable.  Running the  export APP_KEY   statement
from the shell would be good only for the current shell session;  not very ideal
setting up to run in the background, and it would result in the key changing often
- every time you come back to re-run  the command  after a reboot.

I've found the sequence
   php artisan key:generate
   php artisan config:cache
Effective.

(After copying the .env.example  file to  .env   and editing the file for
environment settings, database type, and database creds,   then
pointing the Apache document root to the public/ directory.)